src/main/java/org/opentripplanner/transit/model/site/GroupStop.java

```diff
@@ -79,6 +82,10 @@ public Geometry getGeometry() {
     return geometry;
   }
 
+  public Geometry getEncompassingAreaGeometry() {
```

Comment:
Add doc to this. Should include (if possible) ref to standard doc. I guess this is only in NeTEx, so it should state that.

Reply:
Done.